main.go

```diff
@@ -17,10 +23,44 @@ Complete documentation is available at https://github.com/IBM/openkommander`,
 		Aliases: []string{"openkommander", "kommander", "okm"},
 	}
 
+	serverCmd := &cobra.Command{
+		Use:   "server",
+		Short: "Start the OpenKommander REST API server",
+		Run: func(cmd *cobra.Command, args []string) {
+			startRESTServer()
+		},
+	}
+
+	rootCmd.AddCommand(serverCmd)
 	commands.RegisterCommands(rootCmd)
 
 	if err := rootCmd.Execute(); err != nil {
 		fmt.Println(err)
 		os.Exit(1)
 	}
 }
+
+func startRESTServer() {
+	router := api.NewRouter()
+	server := &http.Server{
+		Addr:    ":8080",
```

Comment:
can you add this port to a config file? `8080` is a very common port and there might be conflicts on the host os when you start OK